Add an actual `data-bs-touch="false"` example in the carousel docs

patrickhlauke opened the pull request:
this can also make it easier to actually check/debug any issues we may have with this data attribute

patrickhlauke committed:
Add an actual `data-touch="false"` example in the carousel docs

this can also make it easier to actually check/debug any issues we may have with this data attribute

rohit2sharma95 commented on site/content/docs/5.0/components/carousel.md:
```diff
@@ -206,6 +206,34 @@ Add `data-bs-interval=""` to a `.carousel-item` to change the amount of time to
+By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute.
+
+{{< example >}}
+<div id="carouselExampleControlsNoTouching" class="carousel slide" data-bs-ride="carousel" data-touch="false">
```
`data-touch` does not contain bs namespace here, and attributes without bs namespace are ignored by Bootstrap.

patrickhlauke replied:
aha! perhaps that's all there is to it? (though that wouldn't explain the report from users saying it's also causing issues in 4.5)

rohit2sharma95 replied:
NVM, I forgot it is for V4

patrickhlauke committed:
Add missing -bs-

rohit2sharma95 commented:
So it means the issue can not be reproduced in the main branch? (I can see that touch events are not working when `data-bs-touch` is set to false in the example)

patrickhlauke commented:
> So it means the issue can not be reproduced in the main branch? (I can see that touch events are not working when `data-bs-touch` is set to false in the example)

correct, I edited/added this to the other issue about the supposed bug itself. still would like this as an addition to the docs regardless, as it makes it nice and obvious

patrickhlauke committed:
Merge branch 'main' into patrickhlauke-carousel-touch

patrickhlauke committed:
Remove data-bs-ride attribute from example

patrickhlauke committed:
Correct attribute table explanation and remove slide class from example

looking at the code, it's still `ride`, not `slide`the `if "carousel"` bit was rather unclear/confusing

patrickhlauke committed:
Edits...

getting myself confused now

patrickhlauke committed:
Add data-bs-interval to suppress autoplay

patrickhlauke committed:
Tweak carousel options formatting

XhmikosR commented on site/content/docs/5.0/components/carousel.md:
```diff
@@ -297,15 +325,15 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
     </tr>
     <tr>
-      <td><code>slide</code></td>
+      <td><code>ride</code></td>
```
Wonder why this was `slide` all this time...

patrickhlauke replied:
yes, either I missed something, or we've had a typo in the options all along...

(admittedly it's the first time I've properly looked at our carousels, but still)

XhmikosR replied:
Yeah, it looks like it's a typo AFAICT. Going to merge this later.

patrickhlauke committed:
Merge branch 'main' into patrickhlauke-carousel-touch
